README.md

```diff
@@ -18,13 +20,26 @@ files that are frequently opened and read as a whole. Furthermore, the software
 use case includes frequent look-ups for files in multiple directories when
 search paths are examined.
 
-Content is published into /cvmfs by means of dedicated "release manager
+Content is published into `/cvmfs` by means of dedicated "release manager
 machines". The release manager machines provide a writeable CernVM-FS instance
-by means of a union file system (aufs or overlayfs) on top of the read-only
+by means of a union file system (`aufs` or `overlayfs`) on top of the read-only
```

Comment:
I think we should remove references to aufs from the docs as we edit them. We are close to deprecating AUFS anyway.

```suggestion
by means of a union file system (e.g., `overlayfs`) on top of the read-only
```